modules/aws-emr-iam/README.md

```diff
@@ -56,6 +56,7 @@ This module creates:
 | emr\_ec2\_role\_name | Name of the new IAM role for EMR EC2 instances | `string` | `"tamr_emr_ec2_role"` | no |
 | emr\_service\_iam\_policy\_name | Name for the IAM policy attached to the EMR Service role | `string` | `"tamr-emr-service-policy"` | no |
 | emr\_service\_role\_name | Name of the new IAM service role for the EMR cluster | `string` | `"tamr_emr_service_role"` | no |
+| permissions\_boundary | ARN of the policy that will be used to set the permissions boundary for all IAM Roles created by this module | `string` | `null` | no |
```

Comment:
Let me know if you've been doing these by hand. If so, it might be worth spending a bit of time to get `make terraform/docs` working for your team.

Reply:
Yes, please!